### PR TITLE
feat: WIP upgrade to php 8.1 and alpine

### PR DESCRIPTION
**Pull request by backendArchitect**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
-FROM alpine:3.9
+FROM alpine:3.18
 MAINTAINER Rakshit Menpara <rakshit@improwised.com>
 
-ENV DOCKERIZE_VERSION v0.6.0
+ENV DOCKERIZE_VERSION v0.7.0
```
Comment (munir131) [inline Dockerfile]: @codeArtisanry Move back to version 0.6.1 - We have issue on 0.7.0 https://github.com/jwilder/dockerize/issues/191

File: Dockerfile [hunk 2]
```diff
@@ -38,21 +38,22 @@ RUN set -ex \
...
+    && php81 -r "unlink('composer-setup.php'); unlink('installer.sig');" \
   # Cleanup
   && apk del .build-deps
 
 ##################  INSTALLATION ENDS  ##################
 
 ##################  CONFIGURATION STARTS  ##################
 
-ADD rootfs /
+ADD rootfs   /
```
Comment (munir131) [inline Dockerfile]: @codeArtisanry remove extra space

File: Dockerfile (hunk 1 above)
Comment (backendArchitect) [reply Dockerfile]: Okay

File: Dockerfile (hunk 2 above)
Comment (backendArchitect) [reply Dockerfile]: Okay

File: rootfs/etc/php81/php-fpm.conf [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+; used in logs and stats. There is no limitation on the number of pools which
+; FPM can handle. Your system will tell you anyway :)
+
+; Include one or more files. If glob(3) exists, it is used to include a bunch of
+; files from a glob(3) pattern. This directive can be used everywhere in the
+; file.
+; Relative path can also be used. They will be prefixed by:
+;  - the global prefix if it's been set (-p argument)
+;  - /usr otherwise
+include=/etc/php82/php-fpm.d/*.conf
```
Comment (munir131) [inline rootfs/etc/php81/php-fpm.conf]: @codeArtisanry it should be 
```suggestion
include=/etc/php81/php-fpm.d/*.conf
```

File: rootfs/etc/php81/php-fpm.d/www.conf [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+; instead.
+
+; Note: path INI options can be relative and will be expanded with the prefix
+; (pool, global or /usr)
+
+; Default Value: nothing is defined by default except the values in php.ini and
+;                specified at startup with the -d argument
+;php_admin_value[sendmail_path] = /usr/sbin/sendmail -t -i -f www@my.domain.com
+php_flag[display_errors] = On
+php_admin_value[error_log] = /var/log/php82/$pool.error.log
```
Comment (munir131) [inline rootfs/etc/php81/php-fpm.d/www.conf]: @codeArtisanry it should be 
```suggestion
php_admin_value[error_log] = /var/log/php81/$pool.error.log
```

**Comment by backendArchitect** [comment]
@munir131 
Updated changes as you requested

**Review by munir131 — APPROVED** [review]

**Comment by munir131** [comment]
Closing PR as we don't need to merge in master

**Pull request closed without merging**